tests/container_disk_test.go

```diff
@@ -220,4 +220,90 @@ var _ = Describe("[rfe_id:588][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 			})
 		})
 	})
+
+	Describe("Simulate an upgrade from a version where ImageVolume was disabled to a version where it is enabled", Serial, func() {
+		BeforeEach(func() {
+			v, err := getKubernetesVersion()
+			Expect(err).ToNot(HaveOccurred())
+			if v < "1.33" {
+				// Skip the test if the Kubernetes version is lower than 1.33
+				// ImageVolume won't work for versions < 1.33 because of this bug:
+				// https://github.com/kubernetes/kubernetes/pull/130394
+				// Additionally there is currently no way to enable k8s ImageVolume FG
+				// through kubevirtci. It is enabled by default since 1.33.
+				Skip("this test requires Kubernetes version >= 1.33")
+			}
+		})
+
+		DescribeTable("Migration from a source launcher with the bind mount workaround to a target launcher without the bind mount workaround should succeed when ,", func(vmi *v1.VirtualMachineInstance) {
```

Comment:
nit
```suggestion
		DescribeTable("Migration from a source launcher with the bind mount workaround to a target launcher without the bind mount workaround should succeed when", func(vmi *v1.VirtualMachineInstance) {

```

Reply:
I added it because this description is combined with the Entries descriptions.